src/test/java/task_60_2/LoginTests.java

```diff
@@ -0,0 +1,62 @@
+package task_60_2;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+public class LoginTests {
+    private final String CHROME_DRIVER_PATH = "src/test/resources/chromedriver";
+    private final String URL = "https://mail.yandex.com/";
```

Comment:
Should be moved to utility class. It could be class with name Constants or TestConstants where you will store static data